.github/actions/setup-go/action.yml

```diff
@@ -11,7 +11,7 @@ runs:
     - name: Set up Go
       uses: actions/setup-go@v3
       with:
-        go-version: 1.18
+        go-version: 1.19
```

Comment:
Why are we doing this?

Reply:
Had some errors with capnproto that required a Go version bump